src/features/ISECompatibility.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+import * as vscode from "vscode";
+import { LanguageClient } from "vscode-languageclient";
+import { IFeature } from "../feature";
+
+/**
+ * A feature to implement commands to make code like the ISE and reset the settings.
+ */
+export class ISECompatibilityFeature implements IFeature {
+    private iseCommand: vscode.Disposable;
+    private defaultCommand: vscode.Disposable;
+    private settings = [
+        { section: "workbench.activityBar", setting: "visible", value: false },
```

Comment:
I would suggest `section` => `path`, `setting` => `name` and `value` stays as is